app/src/main/java/com/sun/unsplash_02/screen/home/CollectionAdapter.kt

```diff
@@ -0,0 +1,62 @@
+package com.sun.unsplash_02.screen.home
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.sun.unsplash_02.R
+import com.sun.unsplash_02.data.model.Collection
+import kotlinx.android.synthetic.main.item_collection.view.*
+
+class CollectionAdapter :
+    RecyclerView.Adapter<CollectionAdapter.CollectionViewHolder>() {
+
+    private var rowSelected = -1
+    private var listCollections = mutableListOf<Collection>()
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): CollectionViewHolder =
+        CollectionViewHolder(
+            LayoutInflater.from(parent.context).inflate(
+                R.layout.item_collection,
+                parent,
+                false
+            )
+        )
+
+    override fun onBindViewHolder(holder: CollectionViewHolder, position: Int) {
+        val collection = listCollections[position]
+        holder.bind(collection, position)
+        if (rowSelected == position) {
+            holder.itemView.linearCollection.setBackgroundResource(R.color.dark_gray)
+        } else {
+            holder.itemView.linearCollection.setBackgroundResource(R.color.gray_100)
+        }
+    }
+
+    override fun getItemCount() = listCollections.size
+
+    fun setListCollections(collections: MutableList<Collection>) {
+        this.listCollections = collections
```

Comment:
this dùng làm gì vậy ?

Reply:
Dạ để trỏ tới biến listCollections trong adapter